exercises/concept/high-score-board/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Exercism
```

Comment:
```suggestion
Copyright (c) 2021 Exercism
```

Reply:
This needs to be fixed in the ROOT, and then it propagates via a sync. Probably a separate PR :)